lib/cli.js

```diff
@@ -29,8 +29,8 @@ exports._parse = function (argv, _console, callback) {
       DEFAULT_SLEEP_MS)
     .option('--concurrency <concurrency>',
       'Amount of requests to run in parallel, defaults to ' + DEFAULT_CONCURRENCY, DEFAULT_CONCURRENCY)
-    .option('--usermedium <usermedium>', 'Medium where the CSS will be used (e.g. `all` (service default), `print`, `screen`)')
-    .option('--profile <profile>', 'CSS profile to use for validation (e.g. `css3svg` (service default), `css21`, `svg`)')
+    .option('--usermedium <usermedium>', 'Medium where the CSS will be used (e.g. `all` (service default), `print`, `screen`)') // jscs:disable
```

Comment:
I'm guessing this is for line length, would prefer we silence that rule explicitly. Will update